app/uk/gov/hmrc/breathingspaceifstub/model/PeriodInRequest.scala

```diff
@@ -31,7 +31,7 @@ final case class PostPeriodInRequest(
 
 object PostPeriodInRequest { implicit val format = Json.format[PostPeriodInRequest] }
 
-final case class PostPeriodsInRequest(periods: List[PostPeriodInRequest])
+final case class PostPeriodsInRequest(utr: Option[Int], periods: List[PostPeriodInRequest])
```

Comment:
Maybe rename this now as it doesn't just contain Post Periods anymore?

Reply:
Suggestions? Honestly, I consider the UTR a sort of foreign attribute in this data structure, which at the end is used for creating BS Periods. And this is a stub, so I do not really care too much.